modules/luci-mod-network/htdocs/luci-static/resources/view/network/dhcp.js

```diff
@@ -647,6 +647,26 @@ return view.extend({
 		);
 		o.placeholder = '/etc/dnsmasq.servers';
 
+		s.taboption('forward', form.Flag, 'add_mac',
```

Comment:
This is not a flag. Or, it could use a flag but it must then be interpreted (0=base64, etc). Manpage: `--add-mac[=base64|text] `

So this can be a dropdown, with two pre-defined options: `base64` and `text`, or nothing.

Reply:
Devs @jow- and @rmilecki recently stated a preference for `enabled`. So `enabled` ideally triggers the naked `--add-mac`. But that needs to happen in https://github.com/openwrt/openwrt/pull/14349

Reply:
Yes, if https://github.com/openwrt/openwrt/pull/14349 is updated to support values for add-mac, I will also update this patch.

Reply:
```javascript

o = s.taboption('forward', form.Value, 'add_mac',
	_('Add requestor MAC'),
	_('Add the MAC address of the requestor to DNS queries which are forwarded upstream.'));
o.optional = true;
o.rmempty = true;
o.value('', _('off'));
o.value('enabled', _('Enabled'));
o.value('base64');
o.value('text');
```